packages/design-system/components/native/avatars/ClickableAvatar/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from 'react-native'
+
+import { Theme } from '../../../../styles/native'
+import { SizeStylesOptions } from './types'
+
+export const createStyles = ({ colors }: Theme, { width, height }: SizeStylesOptions) =>
+  StyleSheet.create({
+    container: {
+      alignItems: 'center',
+      borderColor: colors.surface.active,
+      borderRadius: 50,
+      borderWidth: 2,
+      height: height + 4,
+      justifyContent: 'center',
+      overflow: 'hidden',
+      width: width + 4,
```

Comment:
why the + 4 here ?

Reply:
@Ronan-Fernandes there is a border outside the avatar itself with some spacement between it